src/mixs/schema/mixs.yaml

```diff
@@ -14737,7 +14755,286 @@ slots:
+      syntax: ^{PMID}|{DOI}|{URL}$
+
 classes:
+  MisipMims:
+    description: Metagenome or Environmental with SIP
+    title: MISIP-MIMS
+    aliases:
+      - misip
+      - mims
+    is_a: Checklist
+    mixin: true
+    slots:
+      - samp_name
+      - size_frac
+      - lib_screen
+      - ref_db
+      - nucl_acid_amp
+      - lib_size
+      - mid
+      - assembly_name
+      - temp
+      - nucl_acid_ext
+      - samp_size
+      - alt
+      - source_mat_id
+      - samp_vol_we_dna_ext
+      - lib_reads_seqd
+      - rel_to_oxygen
+      - samp_collect_device
+      - number_contig
+      - lib_layout
+      - assembly_qual
+      - ref_biomaterial
+      - project_name
+      - lib_vector
+      - adapters
+      - neg_cont_type
+      - assembly_software
+      - annot
+      - pos_cont_type
+      - feat_pred
+      - env_local_scale
+      - samp_mat_process
+      - sim_search_meth
+      - depth
+      - samp_collect_method
+      - env_medium
+      - samp_taxon_id
+      - geo_loc_name
+      - collection_date
+      - seq_meth
+      - lat_lon
+      - elev
+      - env_broad_scale
+      - tax_class
+      - experimental_factor
+      - associated_resource
+      - sop
+      - isotope
+      - isotopolog
+      - isotopolog_label
+      - isotopolog_approach
+      - gradient_position
+      - gradient_pos_density
+      - gradient_pos_rel_am
+      - sip_method
+      - isotopolog_atom_frac
+      - isotopolog_atom_pos
+      - isotopolog_dose
+      - nucleobase_atom_frac
+      - isotopolog_incu_time
+      - internal_standard
+    slot_usage:
+      adapters:
+        recommended: true
+      alt:
+        recommended: true
+      annot:
+        recommended: true
+      assembly_name:
+        recommended: true
+      assembly_qual:
+        recommended: true
+      assembly_software:
+        recommended: true
+      depth:
+        examples:
+          - value: 10 meter
+        recommended: true
+      elev:
+        recommended: true
+      experimental_factor:
+        recommended: true
+      lib_layout:
+        recommended: true
+      lib_reads_seqd:
+        recommended: true
+      lib_screen:
+        recommended: true
+      lib_size:
+        recommended: true
+      lib_vector:
+        recommended: true
+      mid:
+        recommended: true
+      nucl_acid_amp:
+        recommended: true
+      nucl_acid_ext:
+        recommended: true
+      number_contig:
+        recommended: true
+      samp_collect_device:
+        examples:
+          - value: swab, biopsy, niskin bottle, push core, drag swab [GENEPIO:0002713]
+        recommended: true
+      samp_collect_method:
+        examples:
+          - value: swabbing
+        recommended: true
+      samp_mat_process:
+        recommended: true
+      samp_size:
+        recommended: true
+      sop:
+        recommended: true
+      source_mat_id:
+        recommended: true
+      temp:
+        recommended: true
+      isotope:
```

Comment:
slot usage to a minimum.
If these slots are required or recommended for **all classes** using them, assert this requirement on the slot not as slot_usage.
Are there any classes where gradient_position density would be applied, but not required? -- ATM, no